frontend/src/metabase/parameters/utils/mapping-options.js

```diff
@@ -39,15 +43,29 @@ function buildVariableOption(variable) {
   };
 }
 
-export function getParameterMappingOptions(metadata, parameter = null, card) {
-  const options = [];
-  if (card.display === "text") {
-    // text cards don't have parameters
-    return [];
+function buildTextTagOption(tagName) {
+  return {
+    name: tagName,
+    icon: "string",
+    isForeign: false,
+    target: ["text-tag", tagName],
```

Comment:
The `["text-tag", <name>]` format here is pretty arbitrary, but once we decide on it, we have to stick with it. It didn't seem necessary to use a nested structure like `["variable", ["template-tag", <name>]]` since this isn't actually a part of MBQL. But I'm definitely open to other options.